web_src/css/admin.css

```diff
@@ -46,32 +47,37 @@
 .admin dl.admin-dl-horizontal {
   padding: 20px;
   margin: 0;
+  display: flex;
+  flex-wrap: wrap;
 }
 
 .admin dl.admin-dl-horizontal dd {
-  margin-left: 275px;
+  margin-left: auto;
+  width: calc(100% - 245px);
 }
 
 @media (max-width: 767px) {
   .admin dl.admin-dl-horizontal dd {
-    margin-left: 5%;
+    width: calc(100% - 185px);
   }
 }
 
+.admin dl.admin-dl-horizontal .ui.divider {
+  width: 100%;
```

Comment:
By default the `divider` should have `width: 100%`?

Reply:
I put this specifically to fix an issue with invisible divider, let me re-try removing it.

Reply:
Yes, divider disappears when the rule is removed. I think this divider can not work in such a wrapping flexbox layout, so it needs the rule.

Reply:
A comment would be helpful, and generally LGTM

Reply:
Comment added.